lib/pyraf/tests/utils.py

```diff
@@ -4,8 +4,11 @@
 from astropy.utils.data import get_pkg_data_contents
 
 IS_PY2 = sys.version_info < (3, 0)
+DATA_DIR = os.path.join(os.path.dirname(__file__), 'data')
```

Comment:
Why is this necessary? Can't you just use `get_pkg_data_filename` from `astropy.utils.data`? See L4.